docs/reference/monitoring/collecting-monitoring-data.asciidoc

```diff
@@ -0,0 +1,213 @@
+[role="xpack"]
+[testenv="gold"]
+[[collecting-monitoring-data]]
+=== Collecting monitoring data
+++++
+<titleabbrev>Collecting monitoring data</titleabbrev>
+++++
+
+If you enable the Elastic {monitor-features} in your cluster, you can 
+optionally collect metrics about {es}. By default, monitoring is enabled but 
+data collection is disabled. 
+
+This method involves sending the metrics to the monitoring cluster by using 
+exporters. For an alternative method, see <<configuring-metricbeat>>.
+
+NOTE: If you want to collect monitoring data from sources such as Beats and {ls}
+and route it to a monitoring cluster, you must follow this method. You cannot
+use {metricbeat} to ship the monitoring data for those products yet.
+
+Advanced monitoring settings enable you to control how frequently data is 
+collected, configure timeouts, and set the retention period for locally-stored 
+monitoring indices. You can also adjust how monitoring data is displayed. 
+
+To learn about monitoring in general, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}]. 
+
+. Configure your cluster to collect monitoring data:
+
+.. Verify that the `xpack.monitoring.enabled` setting is `true`, which is its 
+default value, on each node in the cluster. For more information, see 
+<<monitoring-settings>>. 
+
+.. Verify that the `xpack.monitoring.elasticsearch.collection.enabled` setting 
+is `true`, which is its default value, on each node in the cluster. 
++
+--
+NOTE: You can specify this setting in either the `elasticsearch.yml` on each 
+node or across the cluster as a dynamic cluster setting. If {es} 
+{security-features} are enabled, you must have `monitor` cluster privileges to 
+view the cluster settings and `manage` cluster privileges to change them.
+
+For more information, see <<monitoring-settings>> and <<cluster-update-settings>>.
+--
+
+.. Set the `xpack.monitoring.collection.enabled` setting to `true` on each
+node in the cluster. By default, it is is disabled (`false`). 
++ 
```

Comment:
This setting can also be toggled from the Monitoring UI itself by clicking the "Turn on Monitoring" button. We allude to this in https://www.elastic.co/guide/en/kibana/master/monitoring-kibana.html as well ("To update the cluster settings in Kibana..."). Should we mention the UI method here as well?

Reply:
Done!